src/main/resources/errors.yaml

```diff
@@ -240,6 +240,16 @@ request-errors:
       
       Note, DeleteMany without a filter truncates the table.
 
+  - scope: FILTER
+    code: INVALID_IN_FILTER_FOR_UPDATE_ONE_DELETE_ONE
+    title: In filter is not allowed for UpdateOne and DeleteOne commands
+    body: |-
+      An UpdateOne or DeleteOne command was issued with $in/$nin filter, filter columns are ${inFilterColumns}.
```

Comment:
can we use below for the message pls, trying to keep the lines that include the variable about what went wrong kind of standard. 

The command used a `$in` or `$nin` filter to select the rows the modify, however these filters can select more than one row.

Update One and Delete One commands can only modify a single rows, and so cannot use `$in` or `$nin`.

The command used an invalid filter on the columns: ${inFilterColumns}.

Resend the request without using the `$in` or `$nin` filter.

Reply:
fixed